### PR TITLE
[CB-167] Fix formatting and indentation on files

### PR DESCRIPTION
**Pull request by LWanjiru**
# JIRA TICKET NAME:
CB-167 - [Fix formatting and indentation on files](https://issues.openmrs.org/browse/CB-167)

## SUMMARY:
Currently, the ObservationComponent.jsx file does not follow the styling best practices.

On fix, the code should follow indentation and spacing best practices.

### CODE REVIEW

File: app/js/components/tabs/tabcomponents/observationComponent.jsx [hunk 1]
```diff
@@ -180,124 +181,131 @@ export default class ObsFilter extends React.Component {
...
+              onChange={this.handleFormChange}
+              name="modifier">
+              <option value="">select a value</option>
+              {answers.map(option)}
+            </select>
           </div>
-      </div> : ""
+        </div> : ""
     );
   }
```
Comment (adamsdenniskariuki) [inline app/js/components/tabs/tabcomponents/observationComponent.jsx:292]: Remove this line

File: app/js/components/tabs/tabcomponents/observationComponent.jsx [hunk 2]
```diff
@@ -327,39 +335,47 @@ export default class ObsFilter extends React.Component {
...
+        </div>
+        <div className="form-group">
+          <div className="col-sm-offset-3 col-sm-6">
+            <button
+              type="submit"
+              className="btn btn-success"
+            >
+            Search
+            </button>
+
```
Comment (adamsdenniskariuki) [inline app/js/components/tabs/tabcomponents/observationComponent.jsx]: Remove this empty line

Comment (LWanjiru) [reply app/js/components/tabs/tabcomponents/observationComponent.jsx]: Resolved.

File: app/js/components/tabs/tabcomponents/observationComponent.jsx (hunk 1 above)
Comment (LWanjiru) [reply app/js/components/tabs/tabcomponents/observationComponent.jsx:292]: Resolved.

**Pull request merged**